extensions/wikia/AdEngine/js/AdContext.js

```diff
@@ -89,6 +89,16 @@ define('ext.wikia.adEngine.adContext', [
 			context.providers.turtle = true;
 		}
 
+		// Monetization Service
+		if (context.forceProviders.monetizationService) {
+			context.providers.monetizationService = true;
+		}
+
+		// Use PostScribe for ScriptWriter implementation when Monetization Service ads are enabled
+		if (context.providers.monetizationService) {
```

Comment:
We need to move this logic to backend (setting the wgUsePostScribe var to true if Monetization Service is in use)